### ISSUE TITLE
Change Hover Color from Green to Orange

### ISSUE THREAD

**Issue by tesseheinricks** [mozmeao/springfield#306]
## Description

Change stroke color on hover from Zilla Green (#00D230) to Firefox Orange (#FF9456)

<img width="1512" alt="Image" src="https://github.com/user-attachments/assets/ba7aecee-64af-469c-95fd-e9dfe3eba05c" />

**Comment by tesseheinricks** [comment]
We could also revert to Off-Black (#161616) for accessibility (really it's just preferred to not mix the orange and green!
